### PR TITLE
[PhotoCellNode]Update formatting of layoutSpec to a flat structure.

### PR DESCRIPTION
**Pull request by jaichaudhry**
Updating the layoutSpecs structure to flat structure to make it more readable.

### CODE REVIEW

File: examples/ASDKgram/Sample/PhotoCellNode.m [hunk 1]
```diff
@@ -113,89 +113,52 @@ - (instancetype)initWithPhotoObject:(PhotoModel *)photo;
...
+  ASStackLayoutSpec *footerStack = [ASStackLayoutSpec verticalStackLayoutSpec];
+  footerStack.spacing = VERTICAL_BUFFER;
+  footerStack.children = @[_photoLikesLabel, _photoDescriptionLabel, _photoCommentsView];
+  
+  ASInsetLayoutSpec *footerInset = [ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsMake(VERTICAL_BUFFER, HORIZONTAL_BUFFER, VERTICAL_BUFFER, HORIZONTAL_BUFFER)
+                                                                          child:footerStack];
+  
+  // Main stack
+  ASStackLayoutSpec *mainStack = [ASStackLayoutSpec verticalStackLayoutSpec];
+  mainStack.children = @[headerStackInset, centerPhoto, footerInset];
```
Comment (nguyenhuy) [inline examples/ASDKgram/Sample/PhotoCellNode.m]: Missing: `mainStack.alignItems = ASStackLayoutAlignItemsStretch;`

File: examples/ASDKgram/Sample/PhotoCellNode.m [hunk 2]
```diff
@@ -113,89 +113,52 @@ - (instancetype)initWithPhotoObject:(PhotoModel *)photo;
...
-       ]
-    ]];
+  // Avatar image with inset
+  _userAvatarImageView.style.preferredSize = CGSizeMake(USER_IMAGE_HEIGHT, USER_IMAGE_HEIGHT);
+  ASInsetLayoutSpec *avatarInset = [ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsMake(HORIZONTAL_BUFFER, 0, HORIZONTAL_BUFFER, HORIZONTAL_BUFFER)
+                                                                          child:_userAvatarImageView];
+  
+  // User and photo location stack
+  ASStackLayoutSpec *userPhotoLocationStack = [ASStackLayoutSpec verticalStackLayoutSpec];
+  userPhotoLocationStack.style.flexShrink = 1.0;
```
Comment (nguyenhuy) [inline examples/ASDKgram/Sample/PhotoCellNode.m]: Missing: `userPhotoLocationStack.alignItems = ASStackLayoutAlignItemsStretch;`

File: examples/ASDKgram/Sample/PhotoCellNode.m [hunk 3]
```diff
@@ -113,89 +113,52 @@ - (instancetype)initWithPhotoObject:(PhotoModel *)photo;
...
+  
+  ASInsetLayoutSpec *headerStackInset = [ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsMake(0, HORIZONTAL_BUFFER, 0, HORIZONTAL_BUFFER)
+                                                                               child:headerStack];
+  
+  // Center photo with ratio
+  ASRatioLayoutSpec *centerPhoto = [ASRatioLayoutSpec ratioLayoutSpecWithRatio:1.0 child:_photoImageView];
+  
+  // Footer stack
+  ASStackLayoutSpec *footerStack = [ASStackLayoutSpec verticalStackLayoutSpec];
+  footerStack.spacing = VERTICAL_BUFFER;
```
Comment (nguyenhuy) [inline examples/ASDKgram/Sample/PhotoCellNode.m]: Missing: `footerStack.alignItems = ASStackLayoutAlignItemsStretch;`

**Comment by jaichaudhry** [comment]
@maicki Done. Please have a look now.

**Comment by appleguy** [comment]
@jaichaudhry thanks for writing this!  I think it's a great idea to have the contrast between techniques.  I will do a followup diff adding a comment to explain that users are welcome to use whichever style they prefer.

**Comment by jaichaudhry** [comment]
@appleguy My Pleasure :)

**Pull request merged**